bulk/sensor_bulk_load-AssetRecord.csv

```diff
@@ -1501,6 +1501,8 @@ CGINS-FLORTO-01347,,Sensor,1,3-Wavelength Fluorometer for Glider: FLORT Series O
 CGINS-FLORTO-01348,,Sensor,1,3-Wavelength Fluorometer for Glider: FLORT Series O,,WET Labs,ECO Puck BB3,1348,,,,,Woods Hole Oceanographic Institution
 CGINS-FLORTO-01349,,Sensor,1,3-Wavelength Fluorometer for Glider: FLORT Series O,,WET Labs,ECO Puck BB3,1349,,,,,
 CGINS-HYDGN0-11407,,Sensor,0,Hydrogen Sensor: HYDGN,Hydrogen Sensor,RKI Instruments,M2A transmitters,M2A511407,,,,,
+CGINS-HYDGN0-20009,,Sensor,0,Hydrogen Sensor: HYDGN,Hydrogen Sensor,RKI Instruments,M2A transmitters,3305-00019-00000-20009,,,,,
+CGINS-HYDGN0-20010,,Sensor,0,Hydrogen Sensor: HYDGN,Hydrogen Sensor,RKI Instruments,M2A transmitters,3305-00019-00000-20010,,,,,

```

Comment:
We should be using vendor S/Ns for Hydrogen sensors.

Reply:
right - deleted these new additions.  The vendor s/n UIDs are already in sensor bl